Bottom: fad57335c8ae54a82332f718120ee9917b45f78f
Top:    be4682e06b8f4ea490f71d64e9448d6803472328
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-02-22 00:12:23 +0000

POLYGON DEBUG TEMP


---

diff --git a/src/polygon.c b/src/polygon.c
index 88afa1b..2a48b74 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -1,3 +1,4 @@
+#undef NDEBUG
 /*
  *                            COPYRIGHT
  *
diff --git a/src/polygon1.c b/src/polygon1.c
index 45b8494..fbd0b56 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1,3 +1,5 @@
+#undef NDEBUG
+#define DEBUG 1
 /*
        polygon clipping functions. harry eaton implemented the algorithm
        described in "A Closed Set of Algorithms for Performing Set
@@ -103,7 +105,7 @@ int vect_inters2 (Vector A, Vector B, Vector C, Vector D, Vector S1,
 #undef DEBUG_JUMP
 #undef DEBUG_GATHER
 #undef DEBUG_ANGLE
-#undef DEBUG
+//#undef DEBUG
 #ifdef DEBUG
 #define DEBUGP(...) fprintf(stderr, ## __VA_ARGS__)
 #else
